docs/userguide/packaging/index.md

```diff
@@ -107,15 +110,15 @@ Linux
 
 | File | Links to | Purpose |
 |------------------|------------------|------------------------------------------------------------------------------------|
-| libicuuc.so | libicuuc.so.54.3 | Required for link: Applications compiled with ' -licuuc' will follow this symlink. |
-| libicuuc.so.54 | libicuuc.so.54.3 | Required for runtime: This name is what applications actually link against. |
-| libicuuc.so.54.3 | Actual library | Required for runtime and link. Contains the name 'libicuuc.so.54'. |
+| `libicuuc.so` | libicuuc.so.54.3 | Required for link: Applications compiled with ' -licuuc' will follow this symlink. |
+| `libicuuc.so.54` | libicuuc.so.54.3 | Required for runtime: This name is what applications actually link against. |
+| `libicuuc.so.54.3` | Actual library | Required for runtime and link. Contains the name 'libicuuc.so.54'. |
 
 *This discussion gives
```

Comment:
If this was a "Note" then please use the agreed style for that.

Reply:
Noted.